web/menu.jspf

```diff
@@ -93,11 +93,15 @@ org.opensolaris.opengrok.web.Util"
         name="project" multiple="multiple" size="<%=
         Math.min(15, projectsSize) %>"><%
         SortedSet<String> pRequested = cfg.getRequestedProjects();
-        for ( Group group : ph.getGroups() ) {
+        for (Group group : ph.getGroups()) {
             Set<Project> groupProjects = ph.getAllGrouped(group);
-            if ( groupProjects.size() > 0) {
+            if (groupProjects.size() > 0) {
                 %><optgroup label="<%= group.getName() %>"><%
                 for (Project p : groupProjects) {
+                    if (!p.isIndexed()) {
```

Comment:
I think so. Initially the menu listing can be empty and only group names will be displayed. I think this is better than suppressing empty groups. Alternatively, some count (indexed/not indexed) could be displayed.